libs/platform/src/lib/components/link/link.component.ts

```diff
@@ -10,51 +11,59 @@ import {
     ViewChild
 } from '@angular/core';
 import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { BaseComponent } from '../base';
 
 export type LinkType = 'standard' | 'emphasized';
 export type NavigationTarget = '_blank' | '_self' | '_parent' | '_top' | 'framename';
 const VALID_INPUT_TYPES = ['standard', 'emphasized'];
 
+/**
+ * Platform Link implementation based on the
+ * https://github.com/SAP/fundamental-ngx/wiki/Platform:-Link-Component-Technical-Design
+ * documents.
+ *
+ */
 @Component({
     selector: 'fdp-link',
     templateUrl: './link.component.html',
     styleUrls: ['./link.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class LinkComponent implements OnInit, AfterViewInit {
+export class LinkComponent extends BaseComponent implements OnInit, AfterViewInit {
     emphasized: boolean = false;
     isfocused: boolean = false;
 
     /** Access child element, for checking link content*/
     @ViewChild('link', { read: ElementRef })
     anchor: ElementRef;
 
-    /** Id for the link */
-    @Input()
-    id?: string;
-
-    /** href value to Navigate to */
+    /**
+     * href value to Navigate to. sets href to Native anchor.
+     */
     @Input()
     href: string;
 
-    /** target where navigation will happen, Default=same frame */
+    /**
+     * target where navigation will happen, Default=same frame
+     * sets target to Native anchor.
+     */
     @Input()
     target?: NavigationTarget;
 
-    /** type of link, options standard or Emphasized, Default=standard */
+    /** linktype of link, options standard or Emphasized, Default=standard */
     @Input()
-    type: LinkType = 'standard';
+    linktype?: LinkType = 'standard';
```

Comment:
Use camel-case, change `linktype` to `linkType`.

Reply:
Done